src/tools/lint-docs/src/lib.rs

```diff
@@ -84,7 +84,7 @@ impl Lint {
         for &expected in &["### Example", "### Explanation", "{{produces}}"] {
             if expected == "{{produces}}" && self.is_ignored() {
                 if self.doc_contains("{{produces}}") {
-                    return Err(format!(
+                    return Err(
                         "the lint example has `ignore`, but also contains the {{{{produces}}}} marker\n\
```

Comment:
it seems you forgot to unescape the braces in this string

Reply:
Done https://github.com/rust-lang/rust/compare/a7ab37d0d510da9d9f4a4a93f31aef10c8f277e3..eaa45c8b28468e8ec46bf364eeead996bcc3df8c, thanks!